core/server/web/api/v2/content/middleware.js

```diff
@@ -14,10 +14,8 @@ const shared = require('../../../shared');
  * Authentication for public endpoints
  */
 module.exports.authenticatePublic = [
-    auth.authenticate.authenticateClient,
-    auth.authenticate.authenticateUser,
-    // This is a labs-enabled middleware
-    auth.authorize.requiresAuthorizedUserPublicAPI,
```

Comment:
> // This is a labs-enabled middleware

Where does the labs check live now?

Reply:
The content api is no longer behind labs - it's part of ghost be default now :ghost:

Reply:
Yeah - the way you "disable" it now is to not create content keys/integrations